admin/netdata/patches/003-extra-patch-web_gui_main.js.patch

```diff
@@ -0,0 +1,15 @@
+--- a/web/gui/main.js
```

Comment:
I would like to see that this patch removes Sign in button from main HTML page and who is the author of this patch.

I found this patch here:
- https://sources.debian.org/patches/netdata/1.19.0-3/debian/0011-remove-login.patch/
- https://git.progress-linux.org/distributions/engywuck-backports/packages/netdata/tree/debian/patches/debian/0011-remove-login.patch

Reply:
What sign-in button?
I'll add it in the commit message